dns/opnblock/src/opnsense/mvc/app/controllers/OPNsense/opnblock/Api/SettingsController.php

```diff
@@ -0,0 +1,39 @@
+<?php
+namespace OPNsense\opnblock\Api;
+
+use \OPNsense\Base\ApiControllerBase;
+use \OPNsense\opnblock\opnblock;
+use \OPNsense\Core\Config;
+class SettingsController extends ApiControllerBase
```

Comment:
use ApiMutableModelControllerBase instead. It will reduce the controller code to two lines.